command/base.go

```diff
@@ -60,6 +60,7 @@ type BaseCommand struct {
 	flagOutputCurlString bool
 	flagOutputPolicy     bool
 	flagNonInteractive   bool
+	MuteAddrWarning      bool
```

Comment:
```suggestion
	muteAddrWarning      bool
```

We should err on the side of making fields private, unless there's a reason to export them.

Reply:
Yup, ty!